src/components/Editor.js

```diff
@@ -53,10 +53,9 @@ export default React.createClass({
 
   getInitialState() {
     const decorator = new CompositeDecorator(this.props.decorators);
-
+    this.keyCommandListeners = List(this.props.keyCommandListeners);
```

Comment:
can we move this initialization to a `componentWillMount` lifecycle hook? since it doesn't affect the component's state